.aspect/workflows/config.yaml

```diff
@@ -0,0 +1,9 @@
+tasks:
+  - checkout:
+      update_strategy: rebase
+  - test:
```

Comment:
is your plan to add lint in a follow-up PR? it's already configured in the repo so I think it could be enabled now

Reply:
yes, I'll pass that off to Jesse to add